### ISSUE TITLE
Target new Tab instead of pop-up windows

### ISSUE THREAD

**Issue by Guy-Sowden** [DistributedProofreaders/dproofreaders#405]
Target new tab instead of pop-up windows for when opening proofreading menu, 
pop up windows are now blocked by default in most browsers, and are seen a nuisance for users 

`function newProofWin(winURL){
       nwWin=window.open(winURL,'_blank');
}`

I think there may have to be some minor changes to the layout to adjust  scaling

**Comment by cpeel** [comment]
Yes, we need to revisit this. This code was created well before the advent of tabs.

**Comment by cpeel** [comment]
In Dec 2020 we changed it such that by default, new user profiles have "launch in new window" set to "no". We did not change existing users' profiles, however.

When we have a new proofreading interface I suspect we will remove this option entirely.
